### PR TITLE
Descriptor component 

### LINKED ISSUE #299: Descriptor component
Component to be used on Resource Library to display info about a file to be downloaded.

[Design](https://www.figma.com/file/OWV3OO7QJRp6wG38mco5b6/CR-NPD-ResourceLibrary?node-id=258%3A0)

Screenshot 2020-06-09 at 09 44 44

Fields:
- Date*
- Title*
- Tags (max 3)*
- Description (richText)*
- File - Download icon*
- Category icon*

```
<File
 date=""
 title=""
 tags=[]
 description=node
 file=""
 category=""
/>
```

### PR DESCRIPTION
**Pull request by LouiaR**
Type: feat

Fixes #299

### CODE REVIEW

**Comment by gusliedke** [comment]
@LouiaR Can you minimize SVGs please?

**Comment by LouiaR** [comment]
> @LouiaR Can you optimize SVGs please?

Sure

File: src/components/Molecules/Descriptor/Descriptor.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+`;
+
+/**
+ * Descriptor component
+ */
+
+const Descriptor = ({ date, title, categories, tags, children }) => {
+  const tagItems = tags.splice(0, 3).join(' ');
+  const icons = categories.map((category, index) => {
+    const catObj = { ...categories };
```
Comment (gusliedke) [inline src/components/Molecules/Descriptor/Descriptor.js:103]: why do u need catObj? can't you put index straight in `key`?

Comment (LouiaR) [reply src/components/Molecules/Descriptor/Descriptor.js:103]: OOps you're right

**Pull request merged**